web-registry/src/components/molecules/AdditionalProjectMetadata.tsx

```diff
@@ -77,41 +68,20 @@ const AdditionalProjectMetadata: React.FC<MetadataProps> = ({ metadata }) => {
               data={
                 <ArrowLink
                   label={projectId.toString()}
-                  href={
-                    metadata?.['http://regen.network/vcsProjectPage']?.[
-                      '@value'
-                    ]
-                  }
+                  href={metadata?.['regen:vcsProjectPage']?.['@value']}
                 />
               }
             />
           )}
           <LineItem
             label="vcs project type"
-            data={metadata?.['http://regen.network/vcsProjectType']}
+            data={metadata?.['regen:projectType']}
           />
-          <LineItem
-            label="project start date"
-            data={!!startDate && formatDate(startDate)}
-          />
-          <LineItem
-            label="project end date"
-            data={!!endDate && formatDate(endDate)}
-          />
-          {additionalCertification && (
```

Comment:
just a note that we should make sure to not have those as part of the projects metadata anymore

Reply:
Yep. Removed. And it is being updated in standards repo too.